src/index.html

```diff
@@ -1,16 +1,45 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      >
+      <label class="search-form__label search-form__label--big">
+        <input
+          class="search-form__input search-form__input--big"
+          type="text" name="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          >
```

Comment:
```suggestion
        >
```
![image](https://user-images.githubusercontent.com/99652389/178420327-72136b12-26d0-4e1a-b78b-5d332cef4858.png)